timeseries/tests/smoketests/test_all_models.py

```diff
@@ -166,7 +165,7 @@ def test_all_models_can_handle_all_covariates(
         target=TARGET_COLUMN,
         prediction_length=prediction_length,
         known_covariates_names=known_covariates_names if len(known_covariates_names) > 0 else None,
-        eval_metric=eval_metric,
+        eval_metric="WQL",
```

Comment:
Can we add a test to `models/test_mlforecast.py` that ensures that `RecursiveTabular` and `DirectTabular` work with all covariate combinations for both point & quantile eval_metrics? These models change their behavior depending on whether point or quantile metric is used, so I think we should have a test that validates their behavior for all covariate + metric combinations.